Upstream-Status: Inappropriate [other]
Upstream is not making further releases of this software.

Signed-off-by: Scott Garman <scott.a.garman@intel.com>

# Remove the requirement to link with libwrap
# Patch origin: Wind River

Index: nfs-server-2.2beta47/haccess.c
===================================================================
--- nfs-server-2.2beta47.orig/haccess.c	1999-04-08 08:47:19.000000000 -0400
+++ nfs-server-2.2beta47/haccess.c	2006-08-07 17:05:31.868221639 -0400
@@ -79,8 +79,12 @@
 		clients[hash] = hp;
 
 		hp->clnt_addr = addr;
+#ifdef USE_TCP_WRAPPERS
 		hp->status = hosts_ctl(rpcprog, "unknown",
 					inet_ntoa(addr), "root");
+#else
+		hp->status = 1;
+#endif
 		nrhosts++;
 	}
 
